### ISSUE TITLE
Lighthouse extraction failures 

### ISSUE THREAD

**Issue by MRuecklCC** [openeduhub/metalookup#116]
Sometimes lighthouse still fails to extract the accessibility. Eventually this is the same problem as #115. As we use `str` (instead of `HttpUrl`) for URLs in the Lighthouse API data model.

On the other hand, we use `HttpUrl` in the extractor data model, so no invalid URL that comes through the `/extract` endpoint should be able to reach the Lighthouse API.

AC: 
- Replace `str` with `HttpURL` in Lighthouse API `Input` data model
- Improve logging to contain the URL where lighthouse failed.

Log sample:
```
extractor_1   | 2022-06-21 14:57:47,928 INFO metalookup.app.api Received request for https://www.bpb.de/internationales/weltweit/menschenrechte/158025/hinrichtungen
extractor_1   | 2022-06-21 14:57:59,260 INFO metalookup.core.metadata_manager Built WebsiteData object in 11.33s.
...
[other extractor logs]
... 
extractor_1   | 2022-06-21 14:58:01,348 INFO metalookup.core.metadata_manager Extracted Advertisement in  2.09s.
extractor_1   | 2022-06-21 14:58:06,414 ERROR metalookup.core.metadata_manager Failed to extract accessibility
extractor_1   | Traceback (most recent call last):
extractor_1   |   File "/usr/local/lib/python3.10/site-packages/metalookup/core/metadata_manager.py", line 113, in run_extractor
extractor_1   |     stars, explanation, extra_data = await extractor.extract(site=site, executor=self.process_pool)
extractor_1   |   File "/usr/local/lib/python3.10/site-packages/metalookup/features/accessibility.py", line 50, in extract
extractor_1   |     scores = await asyncio.gather(
extractor_1   |   File "/usr/local/lib/python3.10/site-packages/metalookup/features/accessibility.py", line 92, in _execute_api_call
extractor_1   |     raise RuntimeError(f"Request to lighthouse failed with {response.status}: {await response.text()}")
extractor_1   | RuntimeError: Request to lighthouse failed with 502: {"detail":"Lighthouse process exited with code 1. Standard Error: Runtime error encountered: Lighthouse was unable to reliably load the page you requested. Make sure you are testing the correct URL and that the server is properly responding to all requests. (Status code: 403)\n"}
extractor_1   | INFO:     127.0.0.1:59378 - "POST /extract?extra=true HTTP/1.1" 200 OK
```
